static/app/components/replays/replayController.tsx

```diff
@@ -9,6 +9,9 @@ import {IconArrow, IconPause, IconPlay, IconRefresh, IconResize} from 'sentry/ic
 import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
 
+import {StyledButton} from '../dropdownButtonV2';
+import CompactSelect from '../forms/compactSelect';
```

Comment:
These should probably be imported as `sentry/components/xxx`

Reply:
Done